lectures/10.HPC_and_parallel_compute/environments/lecture10_env.yaml

```diff
@@ -0,0 +1,16 @@
+name: lecture10_env
```

Comment:
can you name this more specifically? (the ordering of the lectures may change year to year)

Reply:
renamed and refactored in all bash scripts